Documentation/spelling_wordlist.txt

```diff
@@ -417,6 +417,7 @@ hashtable
 hashtables
 healthChecking
 healthPort
+helm install
```

Comment:
nit: shouldn't this be:
```suggestion
helm
```
?

Reply:
Yeah, apologies for the noise. I did not read the doc lint correctly and assumed it was a spell-checking issue, when in fact I did not use proper rst syntax (: